NARFO_BE/NARFO_BE/Models/User.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace NARFO_BE.Models
+{
+    public partial class User
+    {
+        public int UserId { get; set; }
+        public string Username { get; set; }
+        public string DisplayName { get; set; }
+        public string SalesRep { get; set; }
+        public string Name { get; set; }
+        public string Lastname { get; set; }
+        public DateTime? Registered { get; set; }
```

Comment:
Registered is a type DateTime so did u mean RegisterDate?

Reply:
It is the time when the user actually registered for Membership status

Reply:
It's the clients database we're using.... all this corresponds to those tables